gridcomps/ExtData2G/ExtDataGridCompNG.F90

```diff
@@ -1360,7 +1360,7 @@ subroutine IOBundle_Add_Entry(IOBundles,item,entry_num,rc)
            io_bundle = ExtDataNG_IOBundle(MAPL_ExtDataLeft, entry_num, current_file, time_index, item%trans, item%fracval, item%file_template, &
                item%pfioCollection_id,item%iclient_collection_id,itemsL,on_tiles,_RC)
            call IOBundles%push_back(io_bundle)
-           call extdata_lgr%info('%a updated L bracket with: %a at time index %i3 ',item%name, current_file, time_index)
+           call extdata_lgr%info('%a updated L bracket with: %a at time index %i5 ',item%name, current_file, time_index)
```

Comment:
Do we maybe just want `i0` so that it is as many digits as are needed?   Or is it important to line up in columns here?

Reply:
No need to line up. I'll change to `i0`